pkg/http/listener_test.go

```diff
@@ -778,12 +777,14 @@ func TestHTTPListenerAcceptParallel(t *testing.T) {
 			if err != nil {
 				t.Fatalf("Test %d: accept: expected = <nil>, got = %v", i+1, err)
 			}
```

Comment:
Don't do these changes in your PR @krishnasrinivas - you should send a separate fix.